src/scripts/apply.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Check CLI config file
+if [[ -n "${TF_PARAM_CLI_CONFIG_FILE}" ]]; then
+    if [[ -f "${TF_PARAM_CLI_CONFIG_FILE}" ]]; then
+        export TF_CLI_CONFIG_FILE=${TF_PARAM_CLI_CONFIG_FILE}
+    else
+        echo "Terraform cli config does not exist: ${TF_PARAM_CLI_CONFIG_FILE}"
+        exit 1
+    fi
+fi
+# 'path' is a required parameter, save it as module_path
+readonly module_path="${TF_PARAM_PATH}"
+export path=$module_path
+
+if [[ ! -d "$module_path" ]]; then
+    echo "Path does not exist: $module_path"
+    exit 1
+fi
+# the following is needed to process backend configs
+if [[ -n "${TF_PARAM_BACKEND_CONFIG_FILE}" ]]; then
+    for file in $(echo "${TF_PARAM_BACKEND_CONFIG_FILE}" | tr ',' '\n'); do
+        if [[ -f "$file" ]]; then
```

Comment:
Hello, just sharing some insight from my work in #41, where we discussed this: 

As documented in https://www.terraform.io/docs/cli/commands/index.html#switching-working-directory-with-chdir , Terraform will 

> change its working directory to the given directory before running the given subcommand. This means that any files that Terraform would normally read or write in the current working directory will be read or written in the given directory instead.

This basically means that this case is no longer needed as Terraform will always search/write in "$module_path" for every command.

This is also a breaking change, as now every path given to Terraform in any command must now be set related to ${module_path} not to ${cwd}. I don't know how you should make it visible for the users, but this will break for people handling their paths based on ${cwd}.

you can check my commit in #41 to see what I found could be changed to handle -chdir https://github.com/CircleCI-Public/terraform-orb/pull/41/commits/98f8a1e85f554df94b06d93915d5372c8826362f